### PR TITLE
Remove alternative 32_power realizations that are not used anymore 

### PR DESCRIPTION
**Pull request by lecfab**
## Purpose of this PR
In module 32_power, only IntC is used (see [conversation](https://mattermost.pik-potsdam.de/rd3/pl/yjs5pg8tujn3tcqgmkh6ouxuce) and [corresponding issue](https://github.com/remindmodel/development_issues/issues/320)).

- removed RLDC and DTcoup realizations
- renamed vm_capDistr vm_demSeOth vm_prodSeOth pm_boundCapCCS pm_cap0  according to their new scope
- updated mrremind, Katarzyna generated new input data: https://github.com/pik-piam/mrremind/pull/527
- updated remind2: 
  - https://github.com/pik-piam/remind2/pull/632
  - https://github.com/pik-piam/remind2/pull/637
  - https://github.com/pik-piam/remind2/pull/639
  - https://github.com/pik-piam/remind2/pull/644

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Spring cleaning
- [x] Refactoring
- [ ] New feature 
- [x] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here:  `/p/tmp/fabricel/powerRealizations/output`
* Comparison of results (what changes by this PR?): nothing, just remove some realizations for 32_power

### CODE REVIEW

File: core/equations.gms [hunk 1]
```diff
@@ -417,47 +417,49 @@ q_limitProd(t,regi,teRe2rlfDetail(teReNoBio(te),rlf))..
...
         ** fm_dataglob("learnExp_wFC",teLearn)
       )
     )$( t.val le 2005 )
-***  special treatment for 2010, 2015: start divergence of regional values by using a
-***  t-split of global 2005 to regional 2020 in order to phase-in the observed 2020 regional
-***  variation from input-data
-  + ( (2020 - t.val)/15 * fm_dataglob("learnMult_wFC",teLearn)
+*** 2005 to 2020: linear transition from global 2005 to regional 2020
+*** to phase-in the observed 2020 regional variation from input-data
+  + ( (2020 - t.val) / (2020-2015) * fm_dataglob("learnMult_wFC",teLearn)
```
Comment (orichters) [inline core/equations.gms:441]: Why did you replace 15 by 2020-2015 = 5?

Comment (lecfab) [reply core/equations.gms:441]: Just to hide some bugs for more fun in the future 🤷 

It's a mistake thanks for digging and finding it!

**Review by orichters — APPROVED** [review]
It looks good from my side, although I still don't understand why this changes the `testOneRegi` behavior as noticed [here](https://github.com/pik-piam/remind2/pull/637). I would prefer if you could wait for @LaviniaBaumstark's approval before merging, maybe she has an idea.

Thanks for the clean-up.

**Comment by lecfab** [comment]
> I still don't understand why this changes the `testOneRegi` behavior as noticed [here](https://github.com/pik-piam/remind2/pull/637). I would prefer if you could wait for @LaviniaBaumstark's approval before merging, maybe she has an idea.

Yes let's do that!

**Comment by lecfab** [comment]
Now that we understand why the [issue with testOneRegi](https://github.com/pik-piam/remind2/pull/637#issuecomment-2304183928) occurred, I think we can merge this PR.
Further cleaning is ongoing in a separate issue about [removing demSeOth](https://github.com/remindmodel/development_issues/issues/338).

What do you think @orichters @fbenke-pik ?

**Comment by orichters** [comment]
Fine from my side.

File: core/equations.gms [hunk 2]
```diff
@@ -156,12 +156,12 @@ q_balSe(t,regi,enty2)$( entySe(enty2) AND (NOT (sameas(enty2,"seel"))) )..
...
       )
     )$( sameas(enty2,"segabio") AND t.val gt 2005 )
-  + sum(prodSeOth2te(enty2,te), vm_prodSeOth(t,regi,enty2,te) )
+  + sum(prodSeOth2te(enty2,te), v_prodSeOth(t,regi,enty2,te) ) !! *** RLDC removal
   + vm_Mport(t,regi,enty2)
   =e=
     sum(se2fe(enty2,enty3,te), vm_demSe(t,regi,enty2,enty3,te))
   + sum(se2se(enty2,enty3,te), vm_demSe(t,regi,enty2,enty3,te))
-  + sum(demSeOth2te(enty2,te), vm_demSeOth(t,regi,enty2,te) )
+  + sum(demSeOth2te(enty2,te), v_demSeOth(t,regi,enty2,te) ) !! *** RLDC removal
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline core/equations.gms:164]: Is `!! *** RLDC removal` intended to conway any information?

Comment (lecfab) [reply core/equations.gms:164]: no, they are anchors for remaining todos (related to [this SeOth issue](https://github.com/remindmodel/development_issues/issues/338) with which you helped me today)

**Pull request merged**